.github/workflows/test-cartesian.yml

```diff
@@ -19,13 +22,13 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.8, 3.9, "3.10"]
-        backends: [internal-cpu, all-cpu]
-        exclude:
-        - python-version: 3.8
-          backends: internal-cpu
-        - python-version: 3.9
-          backends: internal-cpu
+        python-version: ["3.8", "3.9", "3.10"]
+        backends: [internal-cpu, dace-cpu]
+        # exclude:
+        # - python-version: "3.8"
+        #   backends: internal-cpu
+        # - python-version: "3.9"
+        #   backends: internal-cpu
```

Comment:
Once merged we should check that we don't have too many redundant jobs...

Reply:
I've deleted the commented out fragment here because it does not longer apply to the splitting of tests done in cartesian. We can check if there are redundant jobs, but in general CI jobs are now finer grain and executed in parallel, so I don't see obvious redundancies.